fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java

```diff
@@ -362,47 +355,45 @@ public static JSONObject getDefaultParams() throws Exception {
     public static Header[] getDefaultParamsAsHeaders(Header[] pHeaders) throws Exception {
         ArrayList<Header> headers = new ArrayList<Header>();
         JSONObject defaultParams = FH.getDefaultParams();
-        Iterator<String> it = defaultParams.keys();
-        while (it.hasNext()) {
+
+        for (Iterator<String> it = defaultParams.keys(); it.hasNext(); ) {
```

Comment:
You've moved from while() to for(), but without an increment? Why not put that `key = it.next()` in?
This looks like the loop ignores the first item `key = it.next()`.